release-notes/opensearch-release-notes-1.2.0.md

```diff
@@ -0,0 +1,528 @@
+* Remove user from the responses ([#201](https://github.com/opensearch-project/alerting/pull/201))
+
+
+### Opensearch Alerting Dashboards Plugin
+* Fixes flaky test and removes local publishing of plugin dependencies ([#135](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/135))
+* Update copyright notice ([#140](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/140))
+
+
+### Opensearch Anomaly Detection
+* Fix flaky REST IT test ([#259](https://github.com/opensearch-project/anomaly-detection/pull/259))
+* Fixed a bug when door keepers unnecessarily reset their states ([#262](https://github.com/opensearch-project/anomaly-detection/pull/262))
+* Fix task cache expiration bug ([#269](https://github.com/opensearch-project/anomaly-detection/pull/269))
+* Fixed unit test by changing name of method to most up to date ([#287](https://github.com/opensearch-project/anomaly-detection/pull/287))
+* Fix Instant parsing bug in multi category filtering API ([#289](https://github.com/opensearch-project/anomaly-detection/pull/289))
+* Wait for some time to get semaphore when set HC detector task as done ([#300](https://github.com/opensearch-project/anomaly-detection/pull/300))
+* Added switch case for general settings issues ([#305](https://github.com/opensearch-project/anomaly-detection/pull/305))
+
+
+### Opensearch Anomaly Detection Dashboards
+* Fix bug of detector task calls failing if no state index ([#120](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/120))
+* Fix chart range bug of non-HC anomaly and feature charts ([#122](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/122))
+* Support create monitor with custom result index ([#123](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/123))
+* Fix bug of HC historical results not auto-refreshing ([#131](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/131))
+* Fix bug of missing annotations on feature charts ([#136](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/136))
+
+
+### Opensearch Cross Cluster Replication
+* Bugfix: Refactored replication specific translog policy and addressed falky tests ([#236](https://github.com/opensearch-project/cross-cluster-replication/pull/236))
+* Bugfix: Rename translog pruning setting to match 1.1 convention and addressed behavior for translog deletion to be same as 1.1 ([#234](https://github.com/opensearch-project/cross-cluster-replication/pull/234))
+* BugFix: Changes to ensure replication tasks are not failing prematurely ([#231](https://github.com/opensearch-project/cross-cluster-replication/pull/231))
+* Bugfix: Replication task fails to initialize due to state parsing failure ([#226](https://github.com/opensearch-project/cross-cluster-replication/pull/226))
```

Comment:
Should I remove this `BugFix:` and just keep the pull requests?

Reply:
Spelking mistakes - falky vs flaky.

Reply:
Yes, we should follow same format as other plugins

Reply:
Done